lib/checks/001-deprecations.js

```diff
@@ -5,75 +5,125 @@ var _       = require('lodash'),
 
 checkDeprecations = function checkDeprecations(theme, themePath) {
     var checks = [
-            {
-                helperRegEx: /{{pageUrl(\s\S+)?}}/ig,
-                helperName: '{{pageUrl}}',
-                ruleCode: 'GS001-DEPR-PURL'
-            },
-            {
-                helperRegEx: /<meta name=("|')description("|') content=("|'){{meta_description}}("|')/ig,
-                helperName: '{{meta_description}}',
-                ruleCode: 'GS001-DEPR-MD'
-            },
-            {
-                helperRegEx: /{{image}}/g,
-                helperName: '{{image}}',
-                ruleCode: 'GS001-DEPR-IMG'
-            },
-            {
-                helperRegEx: /{{author\.image}}/g,
-                helperName: '{{author.image}}',
-                ruleCode: 'GS001-DEPR-AIMG'
-            },
-            {
-                helperRegEx: /{{post\.image}}/g,
-                helperName: '{{post.image}}',
-                ruleCode: 'GS001-DEPR-PIMG'
-            },
-            {
-                helperRegEx: /{{@blog\.cover}}/g,
-                helperName: '{{@blog.cover}}',
-                ruleCode: 'GS001-DEPR-BC'
-            },
-            {
-                helperRegEx: /{{author\.cover}}/g,
-                helperName: '{{author.cover}}',
-                ruleCode: 'GS001-DEPR-AC'
-            },
-            {
-                helperRegEx: /{{post\.image}}/g,
-                helperName: '{{post.image}}',
-                ruleCode: 'GS001-DEPR-TIMG'
-            },
-            {
-                helperRegEx: /{{@blog\.posts_per_page}}/g,
-                helperName: '{{@blog.posts_per_page}}',
-                ruleCode: 'GS001-DEPR-PPP'
-            },
-            {
-                helperRegEx: /{{content words=("|')0("|')}}/g,
-                helperName: '{{content words="0"}}',
-                ruleCode: 'GS001-DEPR-C0H'
-            },
-            {
-                cssRegEx: /\.archive-template[\s\{]/g,
-                className: '.archive-template',
-                css: true,
-                ruleCode: 'GS001-DEPR-CSS-AT'
-            },
-            {
-                cssRegEx: /\.page[\s\{]/g,
-                className: '.page',
-                css: true,
-                ruleCode: 'GS001-DEPR-CSS-PA'
-            },
-            {
-                cssRegEx: /\.page-template-\w+[\s\{]/g,
-                className: '.page-template-slug',
-                css: true,
-                ruleCode: 'GS001-DEPR-CSS-PATS'
-            }
-        ];
+        {
+            helperRegEx: /{{\s*?pageUrl['"\w\d\s]*?}}/ig,
```

Comment:
This regex also matches {{pageUrlA]]?
But it should only match [[pageUrl something]].